### PR TITLE
Print out healthcare section

### PR DESCRIPTION
**Pull request by lostie**
[Trello](https://trello.com/c/jipJ5uDK/32-5-as-someone-printing-out-a-completed-digital-per-i-want-to-see-a-summary-of-all-the-healthcare-information-in-the-printed-sheet)

Add healthcare assessment content to the move printout.

As an additional part of this PR I've engaged in some extra work to try to simplify the way the assessments information is managed and displayed:

- Destructure assessments information into a more OO approach
- Add an assessment description file that contains all the information
for all the existent assessments: risk and healthcare
- Load the assessment description file and instantiate objects that
contain the information related with the assessment

Right now there's 2 main objects:
- Section - contains a group of questions
- Question - contains all the informations related with a given question

Refactored the print presentation for the assessments to make use of
this new approach.

- Update wording for Assessments summary/printout content

### CODE REVIEW

File: app/models/assessments/question.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      @relevant_answers = data.fetch(:relevant_answers, [])
+      @details = data.fetch(:details, [])
+    end
+
+    def has_dependencies?
+      !dependencies.empty?
+    end
+
+    def relevant_answer?(answer)
+      return false if relevant_answers.empty? || !answer.present?
```
Comment (dan-palmer) [inline app/models/assessments/question.rb]: How about: `answer.blank?`

File: app/models/assessments/question.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      @name = data.fetch(:name)
+      @mandatory = data.fetch(:mandatory, false)
+      @section = data[:section]
+      @dependencies = data.fetch(:dependencies, [])
+      @relevant_answers = data.fetch(:relevant_answers, [])
+      @details = data.fetch(:details, [])
+    end
+
+    def has_dependencies?
+      !dependencies.empty?
```
Comment (dan-palmer) [inline app/models/assessments/question.rb]: How about: `dependencies.present?`

File: app/presenters/print/risk_presenter.rb [hunk 3]
```diff
@@ -1,9 +0,0 @@
-module Print
```
Comment (dan-palmer) [inline app/presenters/print/risk_presenter.rb:1]: 🚶 👋 🚪

**Review by dan-palmer — CHANGES_REQUESTED** [review]
Its probably worth putting something into the Readme discussing the whole assessments shema. Just that its a thing, how & why sorta deal.

File: app/models/assessments/question.rb (hunk 2 above)
Comment (dan-palmer) [reply app/models/assessments/question.rb]: There are a couple more of these as well.

**Comment by lostie** [comment]
♻️

File: app/models/assessments/section.rb [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+    def initialize(name, data = {})
+      @name = name
+      @data = data.deep_symbolize_keys
+      @questions = initialize_questions(@data)
+      @subsections = initialize_subsections(@data)
+    end
+
+    def has_subsections?
+      !subsections.empty?
```
Comment (dan-palmer) [inline app/models/assessments/section.rb]: & another chap here

**Review by dan-palmer — APPROVED** [review]

**Pull request merged**